source/client/client.h

```diff
@@ -636,6 +636,7 @@ extern float scr_con_current;
 extern float scr_conlines;       // lines of console to display
 
 extern ref_export_t re;		// interface to refresh .dll
+extern cgame_export_t *cge; // interface to cgame .dll
```

Comment:
You probably shouldnt make this public?
(Can also make it static again if it is not used anymore)